rkt/prepare.go

```diff
@@ -107,7 +107,9 @@ func runPrepare(cmd *cobra.Command, args []string) (exit int) {
 		return 1
 	}
 
-	if len(flagPodManifest) > 0 && (len(flagPorts) > 0 || flagInheritEnv || !flagExplicitEnv.IsEmpty() || flagStoreOnly || flagNoStore) {
+	if len(flagPodManifest) > 0 && (len(flagPorts) > 0 || flagInheritEnv || !flagExplicitEnv.IsEmpty() || flagStoreOnly || flagNoStore ||
+		(*appsVolume)(&rktApps).String() != "" || (*appMount)(&rktApps).String() != "" || (*appExec)(&rktApps).String() != "" ||
```

Comment:
I didn't see any obvious way to test it via [pflag.Value interface](https://godoc.org/github.com/spf13/pflag#Value) - did I miss anything? An alternative could be adding a custom `isSet()` to each type, to avoid processing complex values, do you prefer that?